src/arrayMethodSort.js

```diff
@@ -5,7 +5,30 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let step;
+    let callback = compareFunction;
+
+    if (!compareFunction) {
+      callback = (curr, prev) => String(curr) < String(prev) ? -1
+        : String(curr) > String(prev) ? 1 : 0;
+    }
```

Comment:
create a separated function `compareByDefault` which you add as a default value (read above)